shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/BaseController.java

```diff
@@ -47,8 +53,39 @@
     @Autowired
     protected MessageSource messageSource;
 
-    @GetMapping
+    @GetMapping(
+        produces = { "application/json" }
+    )
     @ResponseStatus(HttpStatus.OK)
+    @Operation(
+        summary = "Returns all entities",
+        description = "TODO"
+//        security = { @SecurityRequirement(name = "bearer-key") }
+//        content = { @Content(mediaType = "application/json", schema = @Schema(implementation = Class<S>)) }
+    )
```

Comment:
This might be cleaned up.

Reply:
done

Reply:
see https://github.com/terrestris/shogun/pull/595/commits/a94fc4bbc3cc604b1bf09b0ed5a8ebb356ac463b